From 3a54bb485a3a73023314e21b417fbaeb13aa4bd5 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Thu, 15 Aug 2013 11:15:45 +0200
Subject: [PATCH 02/15] CLI: fix ps_show_thread call with bad args count

Tested-By: Jiri Kastner <jkastner@redhat.com>
Signed-off-by: Petr Oros <poros@redhat.com>
Signed-off-by: Jiri Kastner <jkastner@redhat.com>
Signed-off-by: John Kacur <jkacur@redhat.com>
---
 tuna-cmd.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tuna-cmd.py b/tuna-cmd.py
index 64e3ad109d54..5dcc18167d17 100755
--- a/tuna-cmd.py
+++ b/tuna-cmd.py
@@ -227,7 +227,7 @@ def ps_show_thread(pid, affect_children, ps,
 		for tid in ps[pid]["threads"].keys():
 			ps_show_thread(tid, False, ps[pid]["threads"],
 				       has_ctxt_switch_info,
-				       sock_inodes, sock_inode_re)
+				       sock_inodes, sock_inode_re, cgroups)
 			
 
 def ps_show(ps, affect_children, thread_list, cpu_list,
-- 
1.8.3.1

